packages/gatsby-theme-store/src/sdk/buyButton/useBuyButton.ts

```diff
@@ -17,21 +18,36 @@ interface Seller {
 export interface SKU {
   itemId: string
   sellers: Seller[]
+  referenceId: Maybe<Array<{ value: Maybe<string> }>>
+  name: string
   images?: Array<{
     imageUrl: string
   }>
 }
 
+export interface Product {
+  id: string
+  productName: string
+  brand: string
+  categoryTree: Array<{ name: string }>
+  productReference: Maybe<string>
+}
+
 export interface Props {
   sku: Maybe<SKU>
+  product: Maybe<Product>
   quantity: number
   oneClickBuy?: boolean
   openMinicart?: boolean
+  /**
+   * @deprecated This property was deprecated in favor of the new product.productName property
+   */
```

Comment:
Removing this property will make the minicart open with a blank product name.

Reply:
Yeah, but we're replacing it with the `product.productName` property. This is a short-lived deprecation warning since I'll be updating all stores to use the `product` prop and then remove the `productName` prop.

Reply:
I think you can remove this already @icazevedo . That's why we are in alpha :)